src/components/hexGallery/DeskTopGallary.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { GridRow } from "./Styles";
+
+const DeskTopGallary = props => {
+  const { rowOne, rowTwo, rowThree, hexagon } = props;
```

Comment:
See my other message about making these rows an array of arrays and also making this component own it. 
I think the component should be named `DesktopGallery` though.